layouts/index.html

```diff
@@ -0,0 +1,36 @@
+{{ define "main" }}
+<div class="featured-story featured-story-nid-default-featured-story featured-story-light" id="featured-story" data-publish-target="asciidoc">
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12 col-sm-offset-2 col-md-16 col-md-offset-0 featured-story-block featured-story-block-content match-height-item-by-row featured-container"
+                 style="height: 279px;">
+                {{ range where .Site.Pages "Params.block_type" "purpose" }}
```

Comment:
Is there a reason to not hard code this content?

Reply:
I think it's better if the content is not defined in the layout/template.
Arguably, we should also extract the content of the right block... 🤔

Reply:
In most cases, yes, but in some cases, we just hardcode to reduce complexity for rarely changing code or code that will be replaced, in this case by the original featured story potentially. This implementation also has a couple of issues:
1. It creates pages on the live site that will be indexed by bots https://deploy-preview-34--eclipsefdn-asciidoc-wg.netlify.app/index/purpose/
2. If there are multiple pages with these block type parameters, it could print all of them in a row which would bloat out the content.

Reply:
> It creates pages on the live site that will be indexed by bots deploy-preview-34--eclipsefdn-asciidoc-wg.netlify.app/index/purpose

Is it possible to tell Hugo to not generate a page? maybe `ignoreFiles = ["content/index/*"]`?

> If there are multiple pages with these block type parameters, it could print all of them in a row which would bloat out the content.

I guess we could use `first 1`?

Reply:
That would be one solution yes, but keep in mind if we have more than one it might not be the one you'd expect that gets picked as "first". I don't know how pages are retrieved and in what order. Likely alphabetic by file name?

To your question about ignoring pages, we don't use that parameter so I have honestly no clue how it might affect the site render

Reply:
I'm not familiar with Hugo so I don't really know either. I will experiment and let you know 😉

Reply:
So `ignoreFiles = ["content/index/*"]` is working but the pages are completely ignored.
Meaning that `/index/purpose` and `/index/signup` are not generated on the live site but I also cannot get the content in the template 😞 

But we can use `_build.render: never` which does exactly what we want: https://gohugo.io/content-management/build-options/

I moved the pages at the root otherwise Hugo will still generates a listing page for the folder.